be/src/http/action/stream_load.cpp

```diff
@@ -285,6 +285,19 @@ Status StreamLoadAction::_on_header(HttpRequest* http_req, std::shared_ptr<Strea
         }
     }
 
+    if (UNLIKELY((http_req->header(HttpHeaders::CONTENT_LENGTH).empty() &&
```

Comment:
Does master branch also has this problem?

Reply:
yes